code/game/objects/items/weapons/tools/simple_weapons.dm

```diff
@@ -267,6 +267,19 @@
 	else
 		overlays += "[icon_state]_power_off"
 
+/obj/item/tool/sword/katana/vibro
+	name = "artificer vibrokatana"
+	desc = "A modern katana made by a madman painted all over with red and gloss. Just being near it makes your ears ring from it's sharp edge cutting up against the air."
```

Comment:
should have a comma between "madman" and "painted". wrong form of "it's"-- should be "its".